includes/namespace.php

```diff
@@ -45,7 +45,8 @@ function setup_new_site() {
 	$story = $injector->make( Story_Post_Type::class );
 	$story->register();
 	// TODO Register cap to roles within class itself.
-	$story->add_caps_to_roles();
+	$capabilities = $injector->make( User\Capabilities::class );
+	$capabilities->add_caps_to_roles();
```

Comment:
Is this `TODO` still valid? Feels like it would be good to keep the cap registration close to the post type registration.

Reply:
> Is this TODO still valid?

It is. It able removing all logic here and moving all these parts to different classes. 

> Feels like it would be good to keep the cap registration close to the post type registration.

I disagree. The post type class is a mega class that does too much. Register capabilities is something that is used in a number places and needing this whole class to do that is wasteful. Besides, there could be other caps in the future that might do not map to the post type.

Reply:
it is also worth noting, it moves it too the new user namespace. This is more of user action than post type. 

Also see https://github.com/google/web-stories-wp/pull/7276

Reply:
Fixed.